gradle-twirl/src/test/java/play/twirl/gradle/AbstractFunctionalTest.java

```diff
@@ -106,16 +106,19 @@ protected BuildResult build(String gradleVersion, String... args) {
   static Stream<String> gradleVersions() {
     String latest = GradleVersion.current().getVersion();
     // https://docs.gradle.org/current/userguide/compatibility.html
-    if (JavaVersion.current().compareTo(JavaVersion.VERSION_21) >= 0) { // Gradle 8.4+
+    if (JavaVersion.current().compareTo(JavaVersion.VERSION_25) >= 0) { // Gradle 9.1+
       return Stream.of(latest);
     }
-    if (JavaVersion.current().compareTo(JavaVersion.VERSION_17) >= 0) { // Gradle 7.3+
-      return Stream.of("7.6.2", "8.0.2", latest);
+    if (JavaVersion.current().compareTo(JavaVersion.VERSION_21) >= 0) { // Gradle 8.4+
+      return Stream.of("8.4.0", latest);
+    }
+    if (JavaVersion.current().compareTo(JavaVersion.VERSION_17) >= 0) { // Gradle 8.0+
+      return Stream.of("8.0.2", latest);
     }
     // https://docs.gradle.org/current/userguide/scala_plugin.html#sec:configure_zinc_compiler
-    if (getScalaVersion().equals("3")) { // Gradle 7.5+
-      return Stream.of("7.6.2", "8.0.2", latest);
+    if (getScalaVersion().equals("3")) { // Gradle 8.0.2+
+      return Stream.of("8.0.2", latest);
     }
-    return Stream.of("7.1.1", "7.6.2", "8.0.2", latest);
+    return Stream.of("8.0.2", latest);
```

Comment:
I have no idea what I am doing here...
Just changed some versions, but I guess this is not correct....